gateway/src/connectors/pangolin/pangolin.ts

```diff
@@ -43,7 +43,7 @@ export class Pangolin implements Uniswapish {
     this._router = config.routerAddress(network);
     this._ttl = config.ttl;
     this._routerAbi = routerAbi.abi;
-    this._gasLimit = this.avalanche.gasLimit;
+    this._gasEstimate = this.avalanche.gasLimit;
```

Comment:
```suggestion
    this._gasEstimate = this.avalanche.gasEstimate;
```

Reply:
Avalanche is a chain and doesn't have a estimate.
Polygon used avalanche's limit as estimate as far as existing code shows.
Hence, I made those changes accordingly.